sites/fast-website/src/app/components/side-navigation/side-navigation.ts

```diff
@@ -4,6 +4,11 @@ import {
     communityContentPlacementData,
 } from "../../data/community.data";
 
+export interface sectionData {
```

Comment:
More descriptive svp.  Maybe value could be "intersectionRatio" to describe what it is? sectionData.value could be anything...

Reply:
Thank you very much for the suggestion. I have changes value to intersectionRatio and it is much clearer now.